custom/icds_reports/migrations/sql_templates/create_tables_and_views.sql

```diff
@@ -55,6 +55,9 @@ CREATE OR REPLACE VIEW awc_location_months AS
 	months.month_name AS month_display
   FROM awc_location awc_location
   CROSS JOIN "icds_months" months;
+CREATE INDEX "awc_location_months_indx1" ON "awc_location_months" (aggregation_level, month);
+CREATE INDEX "awc_location_months_indx2" ON "awc_location_months" (aggregation_level, state_site_code, month);
+CREATE INDEX "awc_location_months_indx3" ON "awc_location_months" (aggregation_level, state_id, district_id, block_id, supervisor_id, awc_id, month);
```

Comment:
so in sql the result of comparing null to anything is always false, so the result of null = null is false, in which case that view is currently going to be excluding data for all aggregation levels except for the awc level when all the location ids are there.

if that's the way the data is set up think that view should probably be split into separate views for state level aggregation, district level aggregation, etc. where we'd join only on aggregation level and state id, or aggregation level and district id, etc.